go/tools/wtool/main.go

```diff
@@ -0,0 +1,144 @@
+/*wtool augments your bazel WORKSPACE file with new_go_repository entries
+
+Example Usage: wtool com_github_golang_glog com_google_cloud_go
+will add 2 new_go_repository to your WORKSPACE
+by converting com_github_golang_glog -> github.com/golang/glog
+and so forth and then doing a 'git ls-remote' to get
+the latest commit.
+
+if wtool cannot figure out the bazel -> Go mapping, try
+Other Usage: wtool -asis github.com/golang/glog
```

Comment:
I don't think it would not be a problem for users even if `wtool` always behave `-asis`.
Also if it did so, we could avoid assuming the first two components of the given repository name were host names.

Why is the default mode better than `-asis`?